editor/store/test/reducer.js

```diff
@@ -2298,13 +2298,15 @@ describe( 'state', () => {
 						raw: 'The Excerpt',
 					},
 					status: 'draft',
+					preview_link: 'https://wordpress.org/?p=1',
```

Comment:
Nit: Should this include `&preview=true` for clarity?

Reply:
>Nit: Should this include `&preview=true` for clarity?

Yep, sounds good.